.github/workflows/test.yml

```diff
@@ -3,7 +3,9 @@ on:
   push:
     branches: [ "main" ]
   pull_request:
-    branches: [ "main" ]
+      branches: [ "main" ]
```

Comment:
Why the indentation change on this line? If it's not needed, can we revert the extra spaces added?

Reply:
removed indentation